### ISSUE TITLE
Requests issue on mobile network 

### ISSUE THREAD

**Issue by b-j-m** [omz/Pythonista-Issues#566]
I’m using the requests library to get text from google docs. The request works perfectly on WiFi and on LTE using Verizon or AT&T. But on T-Mobile LTE I get a connection error. System info is below. The same code works perfectly on Pythonista 3 on all networks, so there is clearly some buggy interaction here between T-mobile and pythonista 2. (It does work via a VPN on T-Mobile which is my current workaround.)

The code is simple:
import requests
site = 'https://docs.google.com/spreadsheets/d/(removed for privacy)/gviz/tq?tqx=out:csv&sheet=Sheet1&range=A1:D52'
a=requests.get(site,timeout=10)
print(a.text)

I am using pythonista 2 because this code is part of an app I’ve made within Xcode and I need the code available as an app. 

Any ideas?  I suspect it’s the version of the library in pythonista 2 vs 3.   Is there a way to update the library or else switch the Xcode project to python 3? 

**System Information**

* Pythonista 2.1.1 (211008), Default interpreter 2.7.12
* iOS 11.4.1, model iPad6,4, resolution (portrait) 1536.0 x 2048.0 @ 2.0

**Comment by cclauss** [comment]
Did you try increasing __timeout=10__?

**Comment by b-j-m** [comment]
Yes, that is the value that I've been using in my production code.

**Comment by cclauss** [comment]
When you increase it to 60, do things start to work on not?

**Comment by b-j-m** [comment]
Tried 60without success. A timeout of 10 does work in v3, just not v2

**Comment by b-j-m** [comment]
No making it 60 does not fix it either
